js/controllers/hotkeys.js

```diff
@@ -31,8 +31,11 @@ function HotkeysController(windowController, tabs, editor, settings, analytics)
 };
 
 /**
+ * Handles hotkey combination if present in keydown event.
  * Some hotkeys are handled by CodeMirror directly. Among them:
  * Ctrl-C, Ctrl-V, Ctrl-X, Ctrl-Z, Ctrl-Y, Ctrl-A
+ * @param {Event} e The keydown event
```

Comment:
`@param {!Event} e The keydown event`

The Event should never be non-null so I think you should have the `!` at the beginning.

Reply:
Done. Another thing I need to get better at spotting!